Bottom: a8441704c96d34560a9a39746da420f6c967f61c
Top:    41a20f81abcbfe100101265f972e151e97c0af1a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:55:59 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
